recipes/nibabel/run_test.py

```diff
@@ -0,0 +1,14 @@
+import nose
+import sys
+import re
+
+splatform = sys.platform
+if splatform.startswith('win32'):
+    # Installation of data files for script testing a bit rickety on Appveyor:
+    config = nose.config.Config(verbosity=2,
+                                exclude=[re.compile("test_scripts"),
+                                         re.compile("test_dft")])
+else:
+    config = nose.config.Config(verbosity=2)
```

Comment:
I just noticed that the nipy/nibabel repo is not being tested against python 3.5 on this platform...

For example: https://ci.appveyor.com/project/Eric89GXL/nibabel/build/1.0.603

I'll make a PR on nibabel to test on 3.5, so we can see whether we get the same failures there, or if it's something idiosyncratic happening here.